wordpress-mobile/WordPress-iOS
# Images: Fixes an issue with image layout after rotation.

Fixes #10099 
Refs #13757 (fixes?)

The call to `centerImage` was happening before final layout was determined, and was thus using stale geometry. Animating the change along side the transition seems to fix the issue.

To test:
With an iPhone (or iPhone simulator) preview an image either in the Reader or via the Media feature (they both use the WPImageViewController). 
Switch between portrait and landscape orientation.  Confirm the image is correctly positioned.


PR submission checklist:


@frosty could I trouble you for a review?
